tests/wpt/tests/custom-elements/enqueue-custom-element-callback-reactions-inside-another-callback.html

```diff
@@ -1,223 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-<title>Custom Elements: must enqueue an element on the appropriate element queue after checking callback is null and the attribute name</title>
-<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
-<meta name="assert" content="To enqueue a custom element callback reaction, the callback must be checked of being null and whether the attribute name is observed or not">
-<link rel="help" content="https://html.spec.whatwg.org/multipage/custom-elements.html#enqueue-a-custom-element-callback-reaction">
```

Comment:
did you mean to remove this test?

Reply:
Whoops, should be removing the failure expectation...